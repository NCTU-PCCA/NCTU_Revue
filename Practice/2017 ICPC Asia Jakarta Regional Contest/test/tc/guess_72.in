67215395973939 46
<<>><<>><<><><<<>><<>><<<><<>><<<>>>><>><><<<=
