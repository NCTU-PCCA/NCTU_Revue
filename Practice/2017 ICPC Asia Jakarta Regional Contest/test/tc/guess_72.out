33607697986970 16803848993485 8401924496742 12602886745113 14703367869299 13653127307206 13128007026159 13390567166682 13521847236944 13456207201813 13423387184247 13439797193030 13431592188638 13435694690834 13433643439736 13432617814187 13432105001412 13432361407799 13432489610993 13432425509396 13432393458597 13432409483996 13432417496696 13432413490346 13432411487171 13432410485583 13432410986377 13432410735980 13432410610781 13432410673380 13432410704680 13432410689030 13432410681205 13432410677292 13432410679248 13432410680226 13432410680715 13432410680960 13432410680837 13432410680898 13432410680929 13432410680913 13432410680921 13432410680917 13432410680915 13432410680914
